res/values/colors.xml

```diff
@@ -24,6 +24,8 @@
     <color name="owncloud_blue_bright">#00ddff</color>
 
     <color name="list_item_lastmod_and_filesize_text">#989898</color>
+    <color name="actionModeBackground">#757575</color>
+    <color name="actionModeStatusBarBackground">#616161</color>
```

Comment:
Hi @masensio these only need to move to setup.xml _if_ you want to make the multi select action bar colors customizable. When I added the colors I used the "standard" grey colors used in the material design applications I know (e.g. gmail) since they all use the grey colors.

So it really depends on if you want to make this customizable or available to others. If you want them to be in setup.xml, I just put them there, no problem.